code/game/objects/items/crayons.dm

```diff
@@ -105,7 +105,7 @@
 	return
 
 /obj/item/pen/crayon/attack(mob/M as mob, mob/user as mob, var/target_zone)
-	if(M == user)
+	if(M == user && !isipc(user))

```

Comment:
check_has_mouth is probably a better solution here than checking for species.

Reply:
hopefully did that right